sdk/keyvault/azure_security_keyvault_secrets/README.md

```diff
@@ -249,7 +249,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
 ### Delete a secret
 
-`delete_secret` starts a long-running operation to delete a secret previously stored in the Azure Key Vault. You can retrieve the secret immediately without waiting for the operation to complete. When [soft-delete] is not enabled for the Azure Key Vault, this operation permanently deletes the secret.
+`delete_secret` starts a long-running operation to delete a secret previously stored in the Azure Key Vault. You can retrieve the secret immediately without waiting for the operation to complete.
```

Comment:
Please use my recommended text. This still doesn't make sense: "You can retrieve the secret immediately without waiting for the operation to complete."

Reply:
Maybe GitHub is doing something funky here, but it looks like you previously did - though I didn't see it - and now just restored back to what you had.

Reply:
Hey @heaths, 
I think this comment might have been made against an old commit. I removed that old text and used your first suggestion in e0fb752.

I updated the text again in 411eaab to this new suggestion.